src/inspector/view/qml/MuseScore/Inspector/text/TextInspectorView.qml

```diff
@@ -181,25 +183,32 @@ InspectorSectionView {
                     height: 30
 
                     model: [
-                        { iconRole: IconCode.TEXT_ALIGN_LEFT, typeRole: TextTypes.FONT_ALIGN_H_LEFT, title: qsTrc("inspector", "Left") },
-                        { iconRole: IconCode.TEXT_ALIGN_CENTER, typeRole: TextTypes.FONT_ALIGN_H_CENTER, title: qsTrc("inspector", "Center") },
-                        { iconRole: IconCode.TEXT_ALIGN_RIGHT, typeRole: TextTypes.FONT_ALIGN_H_RIGHT, title: qsTrc("inspector", "Right") }
+                        { iconRole: IconCode.TEXT_ALIGN_LEFT, typeRole: TextTypes.FONT_ALIGN_H_LEFT, title: qsTrc("inspector", "Align left"),
+                            description: qsTrc("inspector", "Align left edge of text to reference point") },
+                        { iconRole: IconCode.TEXT_ALIGN_CENTER, typeRole: TextTypes.FONT_ALIGN_H_CENTER, title: qsTrc("inspector", "Align center"),
+                            description: qsTrc("inspector", "Align horizontal center of text to reference point") },
+                        { iconRole: IconCode.TEXT_ALIGN_RIGHT, typeRole: TextTypes.FONT_ALIGN_H_RIGHT, title: qsTrc("inspector", "Align right"),
+                            description: qsTrc("inspector", "Align right edge of text to reference point") }
```

Comment:
These are starting to get messy. Could you write them out on separate lines like this:
```C++
{
    iconRole: IconCode.TEXT_ALIGN_LEFT,
    typeRole: TextTypes.FONT_ALIGN_H_LEFT,
    title: qsTrc("inspector", "Align left"),
    description: qsTrc("inspector", "Align left edge of text to reference point")
},
```
Same with the vertical alignment ones below. The style ones above don't have a description so it's not as bad, but I personally would prefer to see those on separate lines too.